previous-projects.md

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
I think this file was left by mistake when fixing conflicts. This page is now part of home.